### PR TITLE
[T3A2][W11-A1]Tan Wang Leng

### PR DESCRIPTION
**Pull request by yamgent**

### CODE REVIEW

**Comment by yamgent** [comment]
Ready for review.

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -33,4 +33,11 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
         return Optional.ofNullable(relevantPersons);
     }
 
+    /**
+     * @return the feedback to user
+     */
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: consider removing auto-gen  header comments if it adds no value. Generally accessors don't need header comments as their names are self-explanatory.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -125,7 +125,7 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedbackToUser(), DIVIDER);
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: good catch

**Comment by cmkumar87** [comment]
@yamgent Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by yamgent** [comment]
Thanks for your review!

**Pull request closed without merging**